### PR TITLE
Ports sake with a twist

### PR DESCRIPTION
**Pull request by DaxDupont**
[Changelogs]: 

:cl: Dax Dupont
add: You can now get traditional sake to go with your traditional catgirls.
/:cl:

[why]: Why not? Sake seems kinda like staple alcohol we missed.
Also these are kinda codersprite-y.
I took the vodka bottle and redid it partially because the original citadel sprite sucked even more.
@deathride58 did nothing wrong.
Also I took the liberty of picking a random nippon name I liked and renamed the description to be less ...furry.
Also didn't port the random renames since they didn't have unique sprites.

### CODE REVIEW

File: code/modules/food_and_drinks/recipes/drinks_recipes.dm [hunk 1]
```diff
@@ -623,4 +623,12 @@
...
 	required_reagents = list("limejuice" = 2, "capsaicin" = 4) //2 powdered delta (sour), 4 flanergide (spicy)
-	mix_message = "The mixture stings your eyes as it settles."
\ No newline at end of file
+	mix_message = "The mixture stings your eyes as it settles."
+
+/datum/chemical_reaction/sake
+	name = "sake"
+	id = "sake"
+	results = list("sake" = 20)
+	required_reagents = list("rice" = 15, "sugar" = 5)
```
Comment (nicbn) [inline code/modules/food_and_drinks/recipes/drinks_recipes.dm]: Make it use universal enzyme pls

Comment (DaxDupont) [reply code/modules/food_and_drinks/recipes/drinks_recipes.dm]: no

**Comment by Ausops** [comment]
add hiragana and katakana please also kanji

**Comment by pigeonsk** [comment]
Inubashiri isn't furry.

**Comment by DaxDupont** [comment]
@pigeonsk I removed the firefox reference.

File: code/modules/food_and_drinks/drinks/drinks.dm [hunk 2]
```diff
@@ -496,4 +496,4 @@
 	name = "canned air"
 	desc = "There is no air shortage. Do not drink."
 	icon_state = "air"
-	list_reagents = list("nitrogen" = 24, "oxygen" = 6)
+	list_reagents = list("nitrogen" = 24, "oxygen" = 6)
```
Comment (ShizCalev) [inline code/modules/food_and_drinks/drinks/drinks.dm]: why

**Review by ShizCalev — APPROVED** [review]

**Comment by fludd12** [comment]
The name was tetravulpine sake because it's Four Fox Sake, not because it's furry stuff

**Comment by Barhandar** [comment]
It's older than furry, it's Japanese with their fire-slinging kitsunes and other fancy magical lore.

If kitsune stuff is restored, there should also be soju (Korean distilled alcohol) that looks and is named identically to sake, with different description, but is a traitor-only drink with heavier damage - kumiho are basically Korean kitsune except ALWAYS EVIL.

File: code/modules/food_and_drinks/drinks/drinks.dm (hunk 2 above)
Comment (DaxDupont) [reply code/modules/food_and_drinks/drinks/drinks.dm]: I initially put it in the wrong file

**Pull request closed without merging**